plutus-core/plutus-ir/test/PlutusIR/Scoping/Tests.hs

```diff
@@ -1,6 +1,5 @@
-module NamesSpec
-    ( names
-    ) where
+-- editorconfig-checker-disable-file
```

Comment:
Why?

Reply:
I added some `T.`s and it made the lines to long :sob: I should just fix that